docs/metrics-events.md

```diff
@@ -172,7 +188,7 @@ with the following fields:
 |`timestamp`|The time at which the event occurred.|
 |`type`|The name of the event.|
 |`uid`|The user id. An opaque token, HMACed to avoid correlation back to FxA user db.|
-|`device_id`|Optional. The id of the device record.  This *does* correlate back to a record the FxA user db.|
+|`device_id`|Optional. The id of the device record, HMACed to avoid correlation back to the FxA user db.|
```

Comment:
A fix for this is not reflected in the status of the bug at [1], could you please also update that bug with any details?

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1345008